index.html

```diff
@@ -327,26 +394,50 @@ <h3>
           allows API usage in same-origin nested frames but prevents
           third-party content from using the API.
         </p>
-        <p>
-          Third-party usage can be selectively enabled by adding the
-          [^iframe/allow^]`="geolocation"` attribute to an [^iframe^] element:
-        </p>
+        <aside class="addition" id="a6">
+          <span class="marker">Candidate Addition:</span> Add the descripton
+          for "geolocation-approximate" permission and how is it considered
+          with existing "geolocation" permisison.
+        </aside>Third-party usage can be selectively enabled by adding the
+        [^iframe/allow^]`="geolocation"` <ins cite="#a6">or
+        [^iframe/allow^]`="geolocation-approximate"`</ins> attribute to an
+        [^iframe^] element. <ins cite="#a6">Note that if `geolocation` is
```

Comment:
Include double quotes inside the backticks for \`"geolocation"\` and \`"geolocation-approximate"\`